statistics/cmsketch.go

```diff
@@ -54,6 +143,16 @@ func (c *CMSketch) InsertBytes(bytes []byte) {
 	}
 }
 
+// InsertBytesN adds the bytes value into the CM Sketch by n.
+func (c *CMSketch) InsertBytesN(bytes []byte, n uint32) {
```

Comment:
If `c.topnindex != nil`, we should return an error here? 
Or we should merge data in `c.topnindex` into `c.table` and set `c.topnindex=nil` after any insert operation?